Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ['KILT <info@kilt.io>']
 build = 'build.rs'
 edition = '2018'
 name = 'mashnet-node'
-version = '0.23.0'
+version = '0.24.0'
```

Comment:
We should not increase the version number straight away, but wait until we plan to publish a new release.

Reply:
Good catch.

Reply:
But this has to be done to do the new release, right?

Reply:
but we might not want to release right away.